deployment/scripts/install.sh

```diff
@@ -277,11 +277,11 @@ fi
 echo "(Re-)Creating deployed objects"
 if [ "${KEEP_SUPPORT}" != true ]; then
 	oc process logging-support-template | oc delete -f - || :
-	oc delete serviceaccount,service,route --selector logging-infra=support
+	oc delete serviceaccount,service,route,oauthclient --selector logging-infra=support
```

Comment:
Should we delete it by specific name then?  We don't give users a means to rename it at all... or should the oc delete -f - handle that?